table.csv

```diff
@@ -483,6 +483,12 @@ ssz,                            serialization,  0xb501,         draft,     Simpl
 ssz-sha2-256-bmt,               multihash,      0xb502,         draft,     SSZ Merkle tree root using SHA2-256 as the hashing function and SSZ serialization for the block binary
 iscc,                           softhash,       0xcc01,         draft,     ISCC (International Standard Content Code) - similarity preserving hash
 zeroxcert-imprint-256,          zeroxcert,      0xce11,         draft,     0xcert Asset Imprint (root hash)
+varsig,                         varsig,         0xd000,         draft,     Namespace for all not yet standard signature algorithms
+es256k,                         varsig,         0xd0e7,         draft,     ES256K Siganture Algorithm (secp256k1)
+bls-12381-g1-sig,               varsig,         0xd0ea,         draft,     G1 signature for BLS-12381-G2
+bls-12381-g2-sig,               varsig,         0xd0eb,         draft,     G2 signature for BLS-12381-G1
+eddsa,                          varsig,         0xd0ed,         draft,     Edwards-Curve Digital Signature Algorithm
```

Comment:
why the gap of 1? are you anticipating something in between here?

Reply:
The format correlates to the equivalent public key in this table. We're "just" adding `0xd0` to the front of these ones.

| Description      | Type       | Code   |
| ---------------- | ---------- | ------ |
| bls12_381-g2-pub | Public Key | 0xeb   |
| bls12_381-g2-sig | Signature  | 0xd0eb |
| ed25519-pub      | Public Key | 0xed   |
| ed25519-sig      | Signature  | 0xd0ed |

Reply:
Why no `0xd0ec`? Curve25519 is not currently widely used for signatures, with Ed25519 being preferred for this use case (you convert the Curve25519 key pair to Ed25519 and sign with EdDSA).